### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by kriukvlad**
[x] Header height is set in 1 place (for the links)
[x] Content is vertically centered (for any header height)
[x] CSS is used to show all letters in Uppercase (don't type them in HTML)
[x] Logo is an image wrapped with a link
[x] CSS Variable is used for a blue color
[x] Pseudo-element is used for a blue line below the active link
[x] Code follows all the [Code Style Rules ❗️](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,12 +11,106 @@
...
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+              <div class="line"></div>
```
Comment (anastasiiavorobiova) [inline src/index.html:52]:
```suggestion

```
No need to use an additional div here

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,62 @@
...
+  color: var(--blue-color);
+}
+
+.is-active {
+  color: var(--blue-color);
+  margin: 0;
+}
+
+.nav__link.is-active::after {
+  position: absolute;
```
Comment (anastasiiavorobiova) [inline src/style.css:55]:
```suggestion
  position: absolute;
  left: 0;
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

File: readme.md [hunk 3]
```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://kriukvlad.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://kriukvlad.github.io/layout_moyo-header/report/html_report/)
```
Comment (volodymyr-soltys97) [inline readme.md:31]: Add these links to the PR description 
image

File: src/index.html [hunk 4]
```diff
@@ -11,12 +11,105 @@
...
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```
Comment (volodymyr-soltys97) [inline src/index.html:54]: Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav__item">
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,63 @@
+html {
+  --blue-color: rgba(0, 172, 220, 1);
+  --main-bg-color: rgba(255, 255, 255, 1);
```
Comment (volodymyr-soltys97) [inline src/style.css:3]: Create a css variables with root selector 
https://www.w3schools.com/css/css3_variables.asp

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,63 @@
...
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
```
Comment (volodymyr-soltys97) [inline src/style.css:10]: This value is default
```suggestion
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,63 @@
...
+  font-size: 12px;
+  text-transform: uppercase;
```
Comment (volodymyr-soltys97) [inline src/style.css:12]: Move font styles to the html selector, it will be better

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Almost done!
Let's make your code better

**Comment by kriukvlad** [comment]
- [DEMO LINK](https://kriukvlad.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://kriukvlad.github.io/layout_moyo-header/report/html_report/)

[x] Header height is set in 1 place (for the links)
[x] Content is vertically centered (for any header height)
[x] CSS is used to show all letters in Uppercase (don't type them in HTML)
[x] Logo is an image wrapped with a link
[x] CSS Variable is used for a blue color
[x] Pseudo-element is used for a blue line below the active link
[x] Code follows all the [Code Style Rules ❗️](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

**Review by volodymyr-soltys97 — APPROVED** [review]
GJ!

**Pull request closed without merging**